src/Database/Migrations/20210209183016_InitialCreate.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Nvidia.Clara.DicomAdapter.Database.Migrations
+{
+    public partial class InitialCreate : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "ClaraApplicationEntities",
+                columns: table => new
+                {
+                    AeTitle = table.Column<string>(nullable: false),
+                    Name = table.Column<string>(nullable: false),
+                    OverwriteSameInstance = table.Column<bool>(nullable: false, defaultValue: false),
+                    IgnoredSopClasses = table.Column<string>(nullable: true),
+                    Processor = table.Column<string>(nullable: false),
+                    ProcessorSettings = table.Column<string>(nullable: true)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_ClaraApplicationEntities", x => x.AeTitle);
+                });
+
+            migrationBuilder.CreateTable(
+                name: "DestinationApplicationEntities",
+                columns: table => new
+                {
+                    Name = table.Column<string>(nullable: false),
+                    AeTitle = table.Column<string>(nullable: false),
+                    HostIp = table.Column<string>(nullable: false),
+                    Port = table.Column<int>(nullable: false)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_DestinationApplicationEntities", x => x.Name);
+                });
+
+            migrationBuilder.CreateTable(
+                name: "InferenceJobs",
+                columns: table => new
+                {
+                    JobId = table.Column<string>(nullable: false),
+                    PayloadId = table.Column<string>(nullable: false),
+                    JobPayloadsStoragePath = table.Column<string>(nullable: false),
+                    TryCount = table.Column<int>(nullable: false),
+                    State = table.Column<int>(nullable: false)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_InferenceJobs", x => x.JobId);
+                });
+
+            migrationBuilder.CreateTable(
+                name: "InferenceRequests",
+                columns: table => new
+                {
+                    InferenceRequestId = table.Column<Guid>(nullable: false, defaultValue: new Guid("29bb4d8c-f9c5-461f-bb72-d352c73829a0")),
```

Comment:
is this value magic or special? if it is, it should be a `public static readonly Guid MagicGuid = new Guid("29bb4d8c-f9c5-461f-bb72-d352c73829a0");` so it can be reference outside of this method.

Reply:
Generated code